ts/features/design-system/core/DSHapticFeedback.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from "react";
+import ReactNativeHapticFeedback from "react-native-haptic-feedback";
+import { VSpacer } from "../../../components/core/spacer/Spacer";
+import { H2 } from "../../../components/core/typography/H2";
+
+import { IOThemeContext } from "../../../components/core/variables/IOColors";
+import ButtonSolid from "../../../components/ui/ButtonSolid";
+import { DesignSystemScreen } from "../components/DesignSystemScreen";
+
+export const DSHapticFeedback = () => (
+  <IOThemeContext.Consumer>
+    {theme => (
+      <DesignSystemScreen title={"Haptic Feedback"}>
+        <H2 color={theme["textHeading-default"]}>Feedbacks</H2>
```

Comment:
```suggestion
        <H2 color={theme["textHeading-default"]}>Feedback</H2>
```

The noun feedback is uncountable 🤓 💙